README.md

```diff
@@ -6,6 +6,33 @@
 
 ## Overview
 
-A C++ library for interfacing with ABB robot controllers supporting Robot Web Services
+A C++ library for interfacing with ABB robot controllers supporting Robot Web Services (RWS).
 
 See [abb_libegm](https://github.com/ros-industrial/abb_libegm) for a companion library that interfaces with EGM.
+
+### Sketch
+
+The following is a conceptual sketch of how this RWS library can be viewed, in relation to an ABB robot controller as well as the EGM companion library mentioned above. 
+
+![RWS sketch](docs/images/rws_sketch.png)
+
+### Requirements
+
+* RobotWare 6.0 or higher.
+
+### Limitations
+
+This library supports several of the RWS services, but not all of them. Some of the supported features are:
+* Reading/writing of IO-signals.
+* Reading/writing of RAPID data.
+* Starting/stopping/resetting the RAPID program.
+* Subscriptions (i.e. receiving notifications when resources are updated).
+* Uploading/downloading/removing files.
+* Checking controller state (e.g. motors on/off, auto/manual mode and RAPID execution running/stopped).
+* And more.
```

Comment:
This seems a bit vague: the first sentence in this section states "several of the RWS services, but not all of them". Then this item in the list says "and more".

Ideally we should be explicit about what is supported. Would it be possible to include an exhaustive list?

Reply:
Yeah, that is true, I will clarify this.